src/Tribe/Attendees.php

```diff
@@ -935,4 +938,45 @@ public function update_attendee( $attendee, $attendee_data ) {
 		return $provider->update_attendee( $attendee, $attendee_data );
 	}
 
+	/**
+	 * Generate the export URL for exporting attendees
+	 *
+	 * @since TBD
+	 *
+	 *
+	 * @return string Relative URL for the export.
+	 */
+	public function get_export_url (){
```

Comment:
```suggestion
	public function get_export_url() {
```
We need to fix the spacing here.